src/vos/tests/vts_checksum.c

```diff
@@ -86,8 +86,8 @@ cia_idx_get_csum(struct cia_idx *idx, struct dcs_csum_info *infos)
 	return ci_idx2csum(&infos[idx->ci_idx], idx->csum_idx);
 }
 
-void
-csum_for_arrays_test_case(void *const *state, struct test_case_args test)
+static void
+csum_for_arrays_test_case(void *const *state, struct test_case_args *test)
```

Comment:
If not modified, it's best to put to make it

const struct test_case_args *

for documentation purpose and also to ensure some future modification doesn't change expectations.

Reply:
Updated